alkira/resource_alkira_service_infoblox_helper.go

```diff
@@ -167,7 +165,7 @@ func setAllInfobloxResourceFields(d *schema.ResourceData, in *alkira.Infoblox) {
 	d.Set("description", in.Description)
 	d.Set("global_cidr_list_id", in.GlobalCidrListId)
 	d.Set("grid_master", deflateInfobloxGridMaster(in.GridMaster))
-	d.Set("instances", deflateInfobloxInstances(in.Instances))
+	d.Set("instance", deflateInfobloxInstances(in.Instances))
```

Comment:
It will be good to separate this into a separate review. It's much cleaner for reviewing purpose...

Reply:
This is not related to segment_name -> segment_id changes